protocol-samples/http-client-to2-sample/src/main/java/org/fido/iot/sample/To2ClientApp.java

```diff
@@ -74,6 +74,8 @@ private void run(String[] args)
 
     DispatchResult dr = clientService.getHelloMessage();
 
+    clientService.setTo1d(to1dBlob);
```

Comment:
Is this change required?

Reply:
yes that is needed because to1dBlob verification was added